### PR TITLE
Add a new recipe for Mobile Chrome Apps

### PR DESCRIPTION
**Pull request by ragingwind**
I've written a new doc about using Polymer Starter Kit on Mobile Chrome Apps. I wish It could be helpful to other developers. Please feel free to address anything to me.

### CODE REVIEW

**Comment by samccone** [comment]
@chuckh mind looking this over once :)

**Comment by chuckh** [comment]
I did a quick review, it make sense to me. Did not have time to test following the steps. Needs the following done:

- Add link to [Recipes](/docs/README.md/) for this doc.
- Update description in main [README.md](/README.md/) under "Included out of the box:" section.

**Comment by ragingwind** [comment]
I've updated that update readme files and add guide video link of https://www.youtube.com/watch?v=-ifgyobPLVg

**Comment by ragingwind** [comment]
@chuckh I've merged from master to solve conflicts. PTAL

File: docs/mobile-chrome-apps.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# Use Polymer Starter Kit for [Mobile Chrome Apps](https://github.com/MobileChromeApps/mobile-chrome-apps)
+
+## Getting started
+
+Polymer Starter Kit could be fully adapted to Mobile Chrome Apps through mobile-friendly features. Mobile Chrome Apps, is based on Apache Cordova, and requires mobile application SDKs such as Android and iOS. so please make sure that installation development tool by following [installation guide](https://github.com/MobileChromeApps/mobile-chrome-apps/blob/master/docs/Installation.md) of Mobile Chrome Apps. And then, You do some further steps to resolve some of restrictions and configurations to use Polymer Starter Kit on Cordova. Looking for a [guide video](https://www.youtube.com/watch?v=-ifgyobPLVg) to better understand.
+
```
Comment (chuckh) [inline docs/mobile-chrome-apps.md]: You might want to add image with link like below to make the video guide stand out more.

<a href="http://www.youtube.com/watch?feature=player_embedded&v=-ifgyobPLVg" target="_blank">Video Tutorial</a>
Click to play video guide.

**Comment by chuckh** [comment]
@ragingwind for updating. LGTM.  I made one minor suggestion above.

@samccone what do you think?

File: docs/mobile-chrome-apps.md (hunk 1 above)
Comment (samccone) [inline docs/mobile-chrome-apps.md]: oh that is a nice idea @chuckh

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@
 * Unit testing with [Web Component Tester](https://github.com/Polymer/web-component-tester)
 * Optional offline setup through [Platinum](https://elements.polymer-project.org/browse?package=platinum-elements) Service Worker elements
 * End-to-end Build Tooling (including [Vulcanize](https://github.com/Polymer/vulcanize))
-* [Recipes](/docs/README.md/) for ES2015 support, Polymer performance, using Chrome Dev Editor, Deploying to GitHub Pages and Deploying to Firebase
+* [Recipes](/docs/README.md/) for ES2015 support, Polymer performance, using Chrome Dev Editor,  Deploying to GitHub Pages, Deploying to Firebase and Mobile Chrome Apps
```
Comment (samccone) [inline README.md]: Deploying to Firebase,

^ missing comma before and

(AP style will be the death of me [[lol journalism degree]])

> Also note that our rule refers to a “simple” series. In more complex constructions, commas are used before the conjunction.

> The most common example of this exception is when a conjunction is used within the items in a series.

http://www.uwosh.edu/faculty_staff/maguirem/explainer.html

**Comment by ragingwind** [comment]
To make clear commits log, I need to make a new PR because readme file conflict from recently committed at master. :/ Adding images and comma according to guide. This is new one https://github.com/PolymerElements/polymer-starter-kit/pull/556 and I close this issue

**Pull request closed without merging**